components/top-header.js

```diff
@@ -20,8 +20,8 @@ export default function TopHeader ({ sub, cat }) {
 
     if (typeof query.by !== 'undefined') {
       if (query.by === '' ||
-          (what === 'stackers' && (query.by === 'stacked' || !USER_SORTS.includes(query.by))) ||
-          (what === 'territories' && (query.by === 'stacked' || !SUB_SORTS.includes(query.by))) ||
+          (what === 'stackers' && (query.by === 'rewardability' || !USER_SORTS.includes(query.by))) ||
+          (what === 'territories' && (query.by === 'stacking' || !SUB_SORTS.includes(query.by))) ||
           (['posts', 'comments'].includes(what) && (query.by === 'zaprank' || !ITEM_SORTS.includes(query.by)))) {
```

Comment:
I also changed `stacked` to `stacking` and `spent` to `spending` ... I felt like it was clearer.